duckchat/duckchat-api/src/main/java/com/duckduckgo/duckchat/api/DuckChat.kt

```diff
@@ -59,4 +67,16 @@ interface DuckChat {
      * Returns `true` if Duck Chat was ever opened before.
      */
     suspend fun wasOpenedBefore(): Boolean
+
+    /**
+     * Returns the DuckChat AddressBarSettings.
+     */
+    fun getAddressBarSettings(): AddressBarSettings
```

Comment:
we can move this to another class, doesn’t really belong to DuckChat as an api.

Reply:
Moved to separate feature